### ISSUE TITLE
HTTPD docker completely ignored MASS_VHOST_SSL_GEN	to 0

### ISSUE THREAD

**Issue by CristianCojocari** [cytopia/devilbox#830]
I've tried to stop devilbox to auto generate a new pair of ssl certificates on docker-compose down, so I could use my own .crt and .key, the problem is that devilbox completely ignores MASS_VHOST_SSL_GEN=0 from docker-compose.yml.
I tried to bash into the httpd container and do echo $MASS_VHOST_SSL_GEN and it does give me 0, but it still auto generates the certificates.

Is there an actual way I could insert my own certificates, since doing it while the container is still running won't work at all.
Basically I want to stop devilbox from generating new certificates in cert/mass and take a set of certificates that I pass it, either by cp from shared/httpd or by adding them to some folder/config.

**Comment by cytopia** [comment]
Fixed in https://github.com/cytopia/devilbox/releases/tag/v1.10.1
